src/main/java/mate.academy/service/impl/ProductParserImpl.java

```diff
@@ -1,9 +1,11 @@
 package mate.academy.service.impl;
 
 import java.math.BigDecimal;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 import mate.academy.service.ProductParser;
 
+@Component
 public class ProductParserImpl implements ProductParser {
     public static final int ID_POSITION = 0;
```

Comment:
It's recommended to make constant fields `final` to ensure that their values are not accidentally modified.